packages/fx-core/src/common/template-utils/readme.md

```diff
@@ -1,7 +1,14 @@
+## How to scaffold from pre-release templates?
+
+By default, when scaffolding Teams Toolkit downloads latest available templates from [GitHub releases](https://github.com/OfficeDev/TeamsFx/releases).
```

Comment:
Just some comments about English,
"Teams Toolkit downloads the latest stable templates from ... by default. In order to ..."